packages/xeus/src/worker.ts

```diff
@@ -232,6 +230,34 @@ export class XeusRemoteKernel {
     kernelReady(1);
   }
 
+  private _initializeStdin(baseUrl: string, browsingContextId: string): void {
```

Comment:
We should probably make this `protected` here an no-op for `XeusRemoteKernel`? Then move this SW-specific implementation to `comlink.worker.ts`. This leaves the door open to implementing the same in `coincident.worker.ts` later, using `SharedArrayBuffers` communication instead of the service-worker.

Reply:
Done in ac226a9.